src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -409,7 +394,6 @@ const useGroupInfo = (channelId = null) => {
   const leaveGroup = async () => {
     try {
       const response = await channel.removeMembers([profile.myProfile.user_id]);
-      await generateSystemChat('You left this group', profile.myProfile.user_id);
       SimpleToast.show('You left this chat');
       navigation.reset({
         index: 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the removal of the `generateSystemChat` function call might affect the user experience if its functionality isn't replaced elsewhere. Ensure that users are still notified when they leave a group.

```diff
-      await generateSystemChat('You left this group', profile.myProfile.user_id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->